### PR TITLE
fix(docs): Update noirjs_app for 0.23

### PR DESCRIPTION
**Pull request by critesjosh**
# Description

Updates the tutorial to work with v 0.23.0

## Problem\*

Resolves 

## Documentation\*

Check one:
- [ ] No documentation needed.
- [ ] Documentation included in this PR.
- [ ] **[Exceptional Case]** Documentation to be submitted in a separate PR.

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: docs/versioned_docs/version-v0.23.0/tutorials/noirjs_app.md [hunk 1]
```diff
@@ -99,6 +99,24 @@ At this point in the tutorial, your folder structure should look like this:
...
+Add a `vite.config.js` file containing the following:
+
+```js
+import { defineConfig } from 'vite';
+import topLevelAwait from "vite-plugin-top-level-await";
+
+export default defineConfig({
+    plugins: [
+        topLevelAwait({
+            // The export name of top-level await promise for each chunk module
```
Comment (signorecello) [inline docs/versioned_docs/version-v0.23.0/tutorials/noirjs_app.md:111]: I would just remove these comments, they're too descriptive for a tutorial and we don't want for readers to lose focus. Maybe one day we don't need this file at all 🤞

**Review by signorecello — APPROVED** [review]
Just a minor nit

**Comment by Savio-Sou** [comment]
Do we also want this for >0.23.0?

Currently the updated tutorial only shows up under v0.23.0 but not under [dev](https://noir-lang.org/docs/dev/tutorials/noirjs_app) (still describes steps for v0.19.4).

**Comment by critesjosh** [comment]
Created an issue here: https://github.com/AztecProtocol/dev-rel/issues/195

**Pull request merged**